app/models/group.rb

```diff
@@ -60,6 +60,10 @@ def of_ancestors_and_self
   has_many :shared_projects, through: :shared_project_namespaces, class_name: 'Project', source: :project
   has_many :shared_with_groups, through: :shared_with_group_links, source: :group
 
+  def self.ransackable_attributes(_auth_object = nil)
```

Comment:
**nitpick (blocking):** I don't think this is required as it is already set in the namespace model which the group is based on. Can you try removing this and see if it still works as expected.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/749/commits/8ae3cfc0e923ffd7c8defb529798b4391a4b8835